src/views/Shipment.vue

```diff
@@ -0,0 +1,375 @@
+              <Image src="https://cdn.shopify.com/s/files/1/0069/7384/9727/products/test-track.jpg?v=1626255137"/>
+            </div>
+
+            <div class="product-info">
+              <hr />
+
+              <div class="list-item">
+                <ion-item lines="none">
+                  <ion-thumbnail slot="start" class="mobile-only">
+                    <Image src="https://cdn.shopify.com/s/files/1/0069/7384/9727/products/test-track.jpg?v=1626255137" />
+                  </ion-thumbnail>
+                  <ion-label class="ion-text-wrap">
+                    <p class="overline">Brand</p>
+                    Parent product
+                  </ion-label>
+                </ion-item>
+
+                <ion-label class="mobile">
+                  600
+                  <p>{{ $t("shipped") }}</p>
+                </ion-label>
+
+                <ion-label class="mobile">
+                  200
+                  <p>{{ $t("received") }}</p>
+                </ion-label>
+
+                <ion-checkbox />
+
+                <ion-button fill="clear" color="medium" @click="openShippedPopover">
+                  <ion-icon slot="icon-only" :icon="ellipsisVerticalOutline" />
+                </ion-button>
+              </div>
+
+              <ion-list>
+                <ion-list-header>
+                  {{ $t("Items") }}
+                  <hr />
+                </ion-list-header>
+
+                <div class="list-item">
+                  <ion-item lines="none">
+                    <ion-label>
+                      SKU
+                      <p>{{ $t("Color") }}: Color</p>
+                      <p>{{ $t("Size") }}: Size</p>
+                    </ion-label>
+                  </ion-item>
+
+                  <ion-chip class="mobile" outline>
+                    <ion-icon :icon="sendOutline" />
+                    <ion-label>300</ion-label>
+                  </ion-chip>
+
+                  <ion-chip class="mobile" outline>
+                    <ion-icon :icon="checkmarkDoneOutline" />
+                    <ion-label>200</ion-label>
+                  </ion-chip>
+
+                  <ion-checkbox />
+
+                  <ion-button fill="clear" color="medium" @click="openShippedPopover">
+                    <ion-icon slot="icon-only" :icon="ellipsisVerticalOutline"/>
+                  </ion-button>
+                </div>
+
+                <hr />
+
+                <div class="list-item">
+                  <ion-item lines="none">
+                    <ion-label>
+                      SKU
+                      <p>{{ $t("Color") }}: Color</p>
+                      <p>{{ $t("Size") }}: Size</p>
+                    </ion-label>
+                  </ion-item>
+
+                  <ion-chip class="mobile" outline>
+                    <ion-icon :icon="sendOutline" />
+                    <ion-label>300</ion-label>
+                  </ion-chip>
+
+                  <ion-item class="mobile" lines="none">
```

Comment:
No we cant use because without ion-item, icon looks something like this: 
![localhost_8100_shipment](https://user-images.githubusercontent.com/82817616/155291530-39507f2a-8cf4-46bc-938f-9f5f70762774.png)